### PR TITLE
chore: Bundle nodejs runtime using pkg

### PR DESCRIPTION
**Pull request by arajkumar**
This PR bundles nodejs runtime along with lsp for linux, macos and windows 64 bit platforms to eliminate nodejs dependency on intellij and similar platforms.

Also this moves npm package publishing to GH Actions and removes dependency on centos ci.

Note: Other platforms and CPU shall be enabled based on request . e.g BSD, ARM,.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - master
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```
Comment (prashbnair) [inline .github/workflows/release.yml:15]: Should the depth not be 1 as you only need the latest

Comment (arajkumar) [reply .github/workflows/release.yml:15]: This is a release job, it needs all tag details to compute next version.
https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches

**Comment by prashbnair** [comment]
How will this work with environments that have a Node environment like VSCode, will it use the packaged Node version?

**Comment by arajkumar** [comment]
> How will this work with environments that have a Node environment like VSCode, will it use the packaged Node version?

@prashbnair , there will be 2 types of releases,
1. bundled with nodejs 
2. npm registry

VSCode will be using the option #2.

**Review by prashbnair — APPROVED** [review]

**Pull request merged**